File: task_03/C/prime.c
#include <stdio.h>

int main() {
    int i, j, n;
    printf("Enter the last number: ");
    scanf("%d", &n);

    for (i = 2; i <= n; i++) {
        int is_prime = 1;  
        for (j = 2; j * j <= i; j++) {
            if (i % j == 0) {
                is_prime = 0;  
                break;
            }
        }

        if (is_prime) {
            printf("%d\n", i);
        }
    }

    return 0;
}

File: task_03/Ruby/prime.rb

def is_prime?(number)
    return false if number <= 1
  
    (2..Math.sqrt(number)).none? { |i| (number % i).zero? }
  end

  print "Enter a number (n): "
  n = gets.chomp.to_i
  
  
  puts "Prime numbers up to #{n}:"
  (2..n).each { |num| puts num if is_prime?(num) }
  